Donor/Repositories/DonorRepository.cs

```diff
@@ -83,7 +83,86 @@ public async Task<IEnumerable<Entities.Donor>> GetAllDonorsAsync()
 
         public async Task<Entities.Donor> GetDonorByIdAsync(int id)
         {
-            return await _context.Donors.FindAsync(id);
+            return await _context.Donors
+                .Include(d => d.Organs)
+                .FirstOrDefaultAsync(d => d.Id == id);
+        }
+
+
+
+
+        public async Task UpdateDonorAsync(int donorId, Entities.Donor donorUpdate)
+        {
+            using var transaction = await _context.Database.BeginTransactionAsync();
```

Comment:
I don't understand why in an Update method we need to add and Remove properties. The update should be done without all that!